File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { IpGeolocationApi3SDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await IpGeolocationApi3SDK.test()
    equal(null !== testsdk, true)
  })

})

File: rb/test/exists_test.rb
# IpGeolocationApi3 SDK exists test

require "minitest/autorun"
require_relative "../IpGeolocationApi3_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = IpGeolocationApi3SDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
